contracts/proxy/POLYCappedSTOProxy.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.0;
+
+import "./OwnedProxy.sol";
+import "../Pausable.sol";
+import "openzeppelin-solidity/contracts/utils/ReentrancyGuard.sol";
+import "../storage/modules/STO/STOStorage.sol";
+import "../storage/modules/ModuleStorage.sol";
+import "../storage/modules/STO/POLYCappedSTOStorage.sol";
+
+/**
+ * @title POLYCappedSTO module Proxy
+ */
+contract POLYCappedSTOProxy is POLYCappedSTOStorage, STOStorage, ModuleStorage, Pausable, ReentrancyGuard, OwnedProxy {
```

Comment:
There's already an upgradable modules PR ready to be merged. We'll have to refactor this module deployment as well.

Reply:
Yes that was planned after merge. This was another reason for leaving as WIP

Reply:
I've addressed this comment by updated following the merge of the upgradable modules PR. However as you've seen CircleCI still fails when these are in the Experimental folder.